src/Forms/UWP/App.xaml.cs

```diff
@@ -44,7 +44,7 @@ protected override void OnLaunched(LaunchActivatedEventArgs e)
 #if DEBUG
             if (System.Diagnostics.Debugger.IsAttached)
             {
-                DebugSettings.EnableFrameRateCounter = true;
+               // DebugSettings.EnableFrameRateCounter = true;
```

Comment:
Was this intentional?

Reply:
(I did that to take the screenshot, forgot to uncomment after).